Command/UpgradeHelperCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+
+namespace ChameleonSystem\UpgradeHelperBundle\Command;
+
+
+use ChameleonSystem\UpgradeHelperBundle\Parser\Parser;
+use ChameleonSystem\UpgradeHelperBundle\Validator\Validator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class UpgradeHelperCommand extends Command
+{
+    protected static $defaultName = 'chameleon_system:upgrade_helper';
+    /**
+     * @var Parser
+     */
+    private $parser;
```

Comment:
Are we supporting PHP < 7.4 with this bundle? If not, then property types could be used here

Reply:
Also, unless we are supporting <7.2 I see no problem in adding return type hints to all methods

Reply:
chameleon-base claims to be compatible with 7.1. But that should still make it possible to use type hints on all methods

Reply:
So I added type hints to the methods where they were missing except the ones where we extend a parser class which hasn't got a return type declared itself.

Reply:
:ok: (I'll use the :ok: emoji to mark discussions as resolved)